Source/WebCore/animation/AnimationTimeline.h

```diff
@@ -36,6 +37,17 @@ namespace WebCore {
 class AnimationTimelinesController;
 class WebAnimation;
 
+enum class NamedTimelineRange { Normal, Omitted, Cover, Contain, Entry, Exit, EntryCrossing, ExitCrossing };
+struct SingleTimelineRange {
+    NamedTimelineRange range { NamedTimelineRange::Normal };
+    Length offset;
+};
+
+struct TimelineRange {
+    SingleTimelineRange rangeStart;
+    SingleTimelineRange rangeEnd;
```

Comment:
I think we can omit `range` from the property names.
```suggestion
    SingleTimelineRange start;
    SingleTimelineRange end;
```

Reply:
Fixed.